day_7/10_speckled_frogs_extension_2.rb

```diff
@@ -0,0 +1,27 @@
+puts "Enter the number of frogs:"
+print "> "
+count = gets.chomp.to_i
+next_count = count - 1
+
+while count > 2
+   puts "#{count} speckled frogs sat on a log eating some most delicious bugs."
```

Comment:
how could you refactor so that this statement doesn't need to be duplicated below?